policyengine_us/variables/gov/states/me/tax/income/credits/me_child_care_credit/me_child_care_credit.py

```diff
@@ -0,0 +1,42 @@
+from policyengine_us.model_api import *
+
+
+class me_child_care_credit(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "ME child care credit"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://www.maine.gov/revenue/sites/maine.gov.revenue/files/inline-files/22_1040me_sched_a_ff.pdf"  # (y)
+    defined_for = StateCode.ME
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.me.tax.income.credits.child_care
+
+        # Line 1: Total childcare expenses; record percentage of expenses that are regular vs. part of Step 4 child care program
+        expenses = tax_unit("tax_unit_childcare_expenses", period)
+        if expenses == 0:
+            return 0
```

Comment:
I think we want to avoid this because we want to vectorize everything and intermediate return statements make that more difficult (makes total sense to avoid the rest of the calculations though if we were calling the function on one unit as a time)

Reply:
Good point. I had this line not necessarily for efficiency but to avoid returning a NaN - we later divide by expenses. Instead, I've now added the function np.nan_to_num in percentage_paid_regular and percentage_paid_step_4 calculations.